plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObject.class.php

```diff
@@ -36,12 +38,30 @@ public function load()
     $this->count = QubitPdo::fetchColumn($sql, array(QubitInformationObject::ROOT_ID));
   }
 
+  static function loadTerms($options = array())
```

Comment:
This looks like it is just building a lookup list of `term['id'] => term['parent_id']`, in which case I think naming the function "loadTermParentLookups" or "loadTermParentList" or something similar.  From the name "loadTerms()" I would expect it to load term data like names, values, etc.

Reply:
Agreed - implemented in next PR.